packages/core/src/components/Tooltip/index.tsx

```diff
@@ -18,22 +18,24 @@ const EMPTY_TARGET_RECT: ClientRect = {
 };
 
 export type TooltipProps = {
-  /** Width of the tooltip in units. */
-  width?: number;
-  /** What to show in the tooltip. */
-  content: NonNullable<React.ReactNode>;
+  /** Manually override calculated align */
+  align?: 'center' | 'left' | 'right';
```

Comment:
How does center align?

Reply:
Sorry, what do you mean exactly? It centers the tooltip above/below its child

Reply:
Maybe be a bit more explicit, `top-center` and `bottom-center`?

Reply:
@milesj thoughts on making two props instead? e.g. `horizontalAlign` and `verticalAlign`?